### DISCUSSION TITLE
New in pre-reease: swapped `i` and `I` key bindings.

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#4436] [Announcements]
I swapped key bindings: `i` is now `CmdTogglePageInfo`, `I` is `CmdInvertColors`. 

Several people were confused by accidentally typing `i` to invert colors (example: https://github.com/sumatrapdfreader/sumatrapdf/discussions/4421).

`I` is less likely to by pressed accidentally

Available in latest [pre-release](https://www.sumatrapdfreader.org/prerelease) build.

**Comment by thanhminh-vht** [comment]
@kjk 
I really think whatever the binding for `CmdInvertColors` is, there should be a menu item for it in the `View` menu. This helps user discover the feature and also helps them fix when they accidentally enable the feature.
![image](https://github.com/user-attachments/assets/2da03d43-2231-4174-b2f6-b934e7800a10)
